stylix/darwin/default.nix

```
@@ -18,6 +18,6 @@ in
     "${inputs.self}/stylix/palette.nix"
     "${inputs.self}/stylix/pixel.nix"
     "${inputs.self}/stylix/target.nix"
-    (import ../templates.nix inputs)
   ] ++ autoload;
+  config._module.args.inputs = inputs;
```

It might be neater to add an empty line:

```suggestion

  config._module.args.inputs = inputs;
```

Considering we re-export variables under the same internal name, it might be better to ensure consistency between key and values using `inherit`:

```suggestion

  config._module.args = { inherit inputs; };
```